src/api.js

```diff
@@ -670,7 +687,7 @@ API.prototype.declare = function(options, handler) {
   if (options.output && options.output !== 'blob') {
     options.output = this._options.schemaPrefix + options.output;
   }
-  this._entries.push(options);
+  this._entries.unshift(options);
```

Comment:
Made some changes, as far as my understanding of Jonas's suggestion ("probably append it to _entries whenever we read the property") goes. What do you think?